src/Stryker.Core/Stryker.Core/Language/Language.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Stryker.Core
+{
+    public enum Language
+    {
+        Csharp,
+        Fsharp,
+        Undifined,
```

Comment:
Undefined should be the first Enum value so it serves as default(T).

Reply:
yes